### DISCUSSION TITLE
Customized Log  Data

### DISCUSSION THREAD

**Discussion by ferryhtw** [pocketbase/pocketbase#1610] [Q&A]
Hi, 
Is there a way where I can customized Log data?, any clue where I should look?
I noticed that the log does not includes the user who submit the request and I want to includes this information into the Log for login users and guess if the user is not loged in .

**Comment by ganigeorgiev** [comment]
Unfortunately this is not supported at the moment.

There are plans to change the logs implementation in the future to make it more extendable and generic (not just request access logs) so that users can even submit 3rd party app logs, but for now this is a very low priority and will be handled after the eventual "view" collection type and indexes management.

As a workaround, you can create your own "auditLog" collection and update it with any data you want using the available [event request hooks](https://pocketbase.io/docs/event-hooks/). You can find a simple example that reacts to create/update/delete actions in https://github.com/pocketbase/pocketbase/discussions/1281.
